demisto_sdk/commands/coverage_analyze/helpers.py

```diff
@@ -0,0 +1,178 @@
+    if load_old:
+        coverage_obj.load()
+    if combine_from_content_repo:
+        coverage_obj.combine(coverage_files())
+    # uncomment the following for debug purposes
+    # self._cov.set_option('json:pretty_print', True)
+
+    return coverage_obj
+
+
+def coverage_files() -> Iterable[str]:
+    packs_pass = Path('Packs')
+    for cov_path in packs_pass.glob('*/Integrations/*/.coverage'):
+        yield str(cov_path)
+    for cov_path in packs_pass.glob('*/Scripts/*/.coverage'):
+        yield str(cov_path)
+
+
+def get_report_str(coverage_obj) -> str:
+    report_data = io.StringIO()
+    coverage_obj.report(file=report_data)
+    return report_data.getvalue()
+
+
+def percent_to_float(percent: str) -> float:
+    if percent.endswith('%'):
+        percent = percent[:-1]
+    return float(percent)
+
+
+def parse_report_type(report_type_str: Optional[str]) -> List[str]:
+    if report_type_str is None:
+        return []
+
+    allowed_types = ['text', 'html', 'xml', 'json', 'json-min']
+    if report_type_str == 'all':
+        return allowed_types
+
+    report_types = report_type_str.split(',')
+
+    for report_type in report_types:
+        if report_type not in allowed_types:
+            raise InvalidReportType(report_type)
+
+    return report_types
+
+
+class InvalidReportType(Exception):
+
+    def __init__(self, invalid_report_type):
+        self.invalid_report_type = invalid_report_type
+
+    def __str__(self):
+        if self.invalid_report_type == 'all':
+            return 'you cannot use the "all" report type along with other report types'
+        return f"{self.invalid_report_type} is not a valid report type. you can use the following report types as " \
+            "comma separated value for the --report-type argument('text', 'html', 'xml', 'json', 'json-min', 'all')."
+
+
+def export_report(report_call, format, dest):
+
+    logger.info(f'exporting {format} coverage report to {dest}')
+    try:
+        report_call()
+    except coverage.misc.CoverageException as warning:
+        logger.warning(str(warning))
+
+
+class CoverageSummery:
+    def __init__(self, previous_coverage_report_url: str, cache_dir: Optional[str] = None, no_cache: bool = False):
+        self.cache_dir = cache_dir
+        self.url = previous_coverage_report_url
+        self.use_cache = not no_cache
+
+    @staticmethod
+    def create(original_summary_path: str, min_summery_path: str):
+        with open(original_summary_path, 'r') as original_summary_file:
+            original_summary = json.load(original_summary_file)
+
+        min_summery_files = {}
+        original_summary_files = original_summary['files']
+        for py_file_name, py_file_cov_data in original_summary_files.items():
+            min_summery_files[py_file_name] = round(py_file_cov_data['summary']['percent_covered'], 2)
+
+        summary_time = original_summary["meta"]["timestamp"].split(".")[0]
+        min_summery = {
```

Comment:
typo, please fix all places as well
```suggestion
        min_summary = {
```